sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScan.java

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.zapwrapper.scan;
+
+public interface ZapScan {
+
+    void scan();
```

Comment:
The current implementation throws an exception in case of an error during the scan. The exception contains the cause of the error. I wanted to keep this information. I could rework the error handling everywhere by returning some kind of ScanResult, which could contain error information. These ScanResult error information could be processed later on.